src/caret_analyze/common/type_check_decorator.py

```diff
@@ -255,6 +255,24 @@ def _custom_wrapper(*args, **kwargs):
     type_check_decorator = decorator
 
 except ImportError:
+    # NOTE: import pydantic may do before than caret_analyze/common/load_config/init_config()
+    # NOTE: so, this logger is initialized independently.
+    handler = StreamHandler()
+    handler.setLevel(WARN)
+
+    fmt = '%(levelname)-8s: %(asctime)s | %(message)s'
```

Comment:
The warning text was felt to be better in red.
Therefore, it seems better to use ANSI escape sequences and change them as follows.
`fmt = '\033[91m%(levelname)-8s: %(asctime)s | %(message)s\033[0m'`

Reply:
The solution to color the logger will be not implemented because it would be difficult to distinguish it from system errors and warnings.